themes/bootstrap3/templates/layout/layout.phtml

```diff
@@ -50,16 +50,13 @@
         if ($cart->isActive()) {
           $this->headScript()->appendFile("vendor/js.cookie.js");
           $this->headScript()->appendFile("cart.js");
-          $domain = $cart->getCookieDomain();
-          if (!empty($domain)) {
+          if ($domain = $cart->getCookieDomain()) {
             $appendScripts[] = 'VuFind.cart.setDomain("' . $domain . '");';
           }
-          $cookiePath = $cart->getCookiePath();
-          if (!empty($cookiePath)) {
+          if ($cookiePath = $cart->getCookiePath()) {
             $appendScripts[] = 'VuFind.cart.setCookiePath("' . $cookiePath . '");';
           }
-          $cookieSameSite = $cart->getCookieSameSite();
-          if (null !== $cookieSameSite) {
+          if ($cookieSameSite = $cart->getCookieSameSite()) {
```

Comment:
This is a drop in specificity from `!== null`. I don't know if it's a meaningful in the practice of our configs. I guess I should trust the tests.

Reply:
The default value returned is Lax as if someone tries to put an empty value there, it would just be odd.